--- /Library/Motion/lib/motion/project/config.rb	2012-01-23 07:18:14.000000000 -0800
+++ /tmp/config_patch.rb	2012-01-24 21:52:45.000000000 -0800
@@ -194,7 +194,13 @@
     attr_accessor :spec_mode
 
     def spec_files
-      Dir.glob(File.join(specs_dir, '**', '*.rb'))
+      files = Dir.glob(File.join(specs_dir, '**', '*.rb'))
+      files.delete(spec_helper)
+      files
+    end
+
+    def spec_helper
+      File.join(specs_dir, 'spec_helper.rb')
     end
 
     def motiondir
